config-model/src/main/java/com/yahoo/vespa/model/container/search/ContainerSearch.java

```diff
@@ -65,6 +67,11 @@ public ContainerSearch(DeployState deployState, ApplicationContainerCluster clus
         cluster.addSearchAndDocprocBundles();
     }
 
+    private static Collection<String> getSchemasWithGlobalPhase(DeployState state) {
+        return state.rankProfileRegistry().all().stream()
+                .filter(rp -> rp.getGlobalPhase() != null).map(rp -> rp.schema().getName()).toList();
```

Comment:
consider something like `.collect(Collectors.toSet())` instead of `.toList()`

Reply:
Addressed in [4b55e59](https://github.com/vespa-engine/vespa/commit/4b55e59b0bdda6559a40addf0ede434ab955dc07)